package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "loopback-android-getting-started",
+  "version": "1.0.0",
+  "description": "LoopBack mobile application to show developers how to use LoopBack Open Node.js Mobile API Middle tier.",
+  "main": "index.js",
+  "scripts": {
+    "test": ""
```

Comment:
This means that all CI is doing is running `npm install`, which just installs the devDependencies listed below..

Are there actual tests to run?